tests/unit/test-extension-runners/test.firefox-android.js

```diff
@@ -984,37 +984,37 @@ describe('util/extension-runners/firefox-android', () => {
         {
           params: {profilePath: '/fake/dir'},
           expectedMessage: (
-            /Android target does not support --profile-path/
+            (/Android target does not support --profile-path/)
```

Comment:
Why do we need two sets of parens?

Reply:
really no reasons.

I suspect this may have been a jscodeshift doing, maybe a side-effect of turning the transformed AST back into js code (e.g. in a previous version of the transformer I was missing a `quote: 'single'` option and all strings got a double quote in the transformed files).

I'm pretty sure that I removed another similar one that I noticed but in the big diff I missed this one.